### ISSUE TITLE
Allow for configuration of image name

### ISSUE THREAD

**Issue by cbromberg** [localstack/localstack-java-utils#39]
Hi,

first of all kudos to this great project - I am currently in the process of migrating a project that used DynamoDbLocal and ElasticSearch spawned as child processes from Java for testing (basically because our on-prem CI does not allow docker containers) and docker containers for development, which caused a lot of classpath polution and also custom code that somebody (in this case me) to maintain. So I am excited to switch the CI to cloud (Bitbucket pipelines) and in the process switch to localstack spawned from Java as well.
As we are using ElasticSearch I ran into the problem of deferred download of ElasticSearch through the localstack container. Since we use throw-away containers we do not benefit from the in-container caching.
Long story short: If I create a PR to allow for the configuration of the image name to be able to use localstack-full, would you be willing to review that or is that not something you would like in this project.
Of course I would not break existing code
Cheers, Chris

**Comment by whummer** [comment]
Hi @cbromberg , thanks for the feedback.

> If I create a PR to allow for the configuration of the image name to be able to use localstack-full, would you be willing to review that or is that not something you would like in this project.

Sure, sounds like a great suggestion - we'd definitely appreciate a pull request with this change. Thanks for your help!

**Comment by cbromberg** [comment]
Great - will create a PR by tomorrow at the latest

**Comment by whummer** [comment]
Fixed via #40 - thanks!
